src/controllers/help.controller.ts

```diff
@@ -0,0 +1,66 @@
+import { Router, Request, Response } from 'express';
+import { Controller } from '../interfaces';
+import axios from 'axios';
+import MarkdownIt from 'markdown-it';
+
+const md = new MarkdownIt();
+
+const fetchCommands = async (user, repo) => {
+    try {
+        const url = `https://api.github.com/repos/${user}/${repo}/contents/commands.md`;
+        const response = await axios.get(url, {
+            headers: {
+                'Accept': 'application/vnd.github.v3.raw',
+            },
+        });
+        return response.data;
+    } catch (error) {
+        console.error(`Error fetching commands: ${error}`);
+    }
+};
+
+export class HelpController implements Controller {
+    public basepath = '/help';
+
+    public boot(): Router {
+        const router: Router = Router();
+
+        router.get(this.basepath, async (req: Request, res: Response) => {
+            const command = req.body.command ? req.body.command.trim() : null;
+        
+            const commandsMarkdown = await fetchCommands('princerajpoot20', 'api_endpoint');
```

Comment:
we should use the CLI repo's doc, not your own repo 😅, can you change this?

Reply:
Definitely, I am using it for testing locally, which is why it has my repo link😄. I will change it, but regarding the CLI repo's documentation, I think it should have a separate commands.md file in the server-api, as the commands are not the same as those of the CLI.

Reply:
start with the approach 2, although that `usage.md` might not be enough for some scenarios. But use it for your next implementation and test it 🙂 

> If yes, should we add new commands of server-api in this file, as this file does not have all the endpoints of server-api.

We won't add server-api endpoints to this file, since these are docs from the CLI, but go forward with this approach.